back/app/controllers/application_controller.rb

```diff
@@ -31,6 +31,20 @@ def initialize(feature)
 
   rescue_from FeatureRequiredError, with: :feature_required_error
 
+  def self.inherited(base)
+    base.extend(RegisterSerializer)
+  end
+
+  module RegisterSerializer
```

Comment:
Adding this to the `ClassMethods` module didn't work, so this is the best solution I found.